contracts/stellar-example/src/testdata/ensure_data_key_storage_schema_is_unchanged.golden

```diff
@@ -0,0 +1,47 @@
+#[contracttype] enum DataKey
```

Comment:
This is not a very readable golden file. I'd prefer to just have the original enum representation dumped in a golden file to better flag a breaking change